<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Code de la carte  :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCode"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonCaptureCasier">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture Casier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCaptureUtilisateur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture Utilisateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSupprimer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
